spec/system/authenticate_user_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+RSpec.describe "Authentication", type: :system, service: :claims do
+  let(:school) { create(:claims_school, name: "School") }
+  let(:anne) { create(:claims_user, :anne) }
+
+  scenario "As a user who has not signed in" do
+    when_i_visit_claims_schools_details_path
+    then_i_am_unable_to_access_the_page
+  end
+
+  scenario "As a user who has signed in" do
+    given_there_is_an_existing_claims_user_with_a_school_for(anne)
+    when_i_visit_the_sign_in_path
+    when_i_click_sign_in
```

Comment:
not a blocker, but the methods should be in the context of `and_....` you can also use `alias_method` to make `when_i_visit_claims_schools_details_path` to `and_i_visit_claims_schools_details_path`